pkg/iss/config.go

```diff
@@ -18,9 +18,14 @@ import (
 // To obtain real time delays, these need to be multiplied by the period of the ticker provided to the Node at runtime.
 type Config struct {
 
-	// The IDs of all nodes that execute the protocol.
+	// The IDs of all nodes that execute the protocol in the first epoch.
 	// Must not be empty.
-	Membership []t.NodeID
+	InitialMembership []t.NodeID
```

Comment:
Membership in most parts of the codebase is `map[t.NodeID]t.NodeAddress`, maybe we should use another name here?

Reply:
Fixed. (In fact, I needed to use the map of addresses here anyway in the end.)